data/transportation/transit/index.html

```diff
@@ -6,205 +6,189 @@
   display_name: Jessie Pechmann
   email: jpechmann@utah.gov
 tags:
-  - data
+  - transportation
+  - transit
+  - bus
+  - commuter
+  - light rail
+  - uta
+  - stations
 categories: []
 published: true
 date: 2011-11-23 14:16:26 -0700
+UTABusRoutesStops:
+  downloads:
+    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mXy0wcENfR0NWN0k&export=download
+      label: "Bus Routes: File Geodatabase"
+    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mRU9XR3FuTWhDMzA&export=download
+      label: "Bus Routes: Shapefile"
+    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mRTJwVU52c1VCNDA&export=download
+      label: "Bus Stops: File Geodatabase"
+    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mMF9ZUkdiMEpsb3M&export=download
+      label: "Bus Stops: Shapefile"
+  updates:
+    - "Updates to this dataset are published periodically.<br>
```

Comment:
i would prefer if you put the update text in the package content and simply left the date in here.